src/db/Place/place.ts

```diff
@@ -138,81 +156,66 @@ export async function addPlace(placeToAdd: Place): Promise<Place> {
  * */
 export async function updatePlace(id: GooglePlaceID["place_id"]): Promise<Place> {
 	const { _col: placeCol, _connection: placeConn } = await getCollection<Place>("place");
-
 	const { _col: ratingCol, _connection: ratingConn } = await getCollection<Rating>("rating");
 
 	const pipeline = [
-		{
-			$match: {
-				placeID: id,
-				braille: {
-					$exists: 1,
-				},
-				fontReadability: {
-					$exists: 1,
-				},
-				staffHelpfulness: {
-					$exists: 1,
-				},
-				guideDogFriendly: {
-					$exists: 1,
-				},
-				navigability: {
-					$exists: 1,
-				},
-			},
-		},
 		{
```

Comment:
Why did you get rid of this step entirely? Not saying you're wrong, I'm genuinely curious. When I first wrote this code, I was half sure of what I was doing, so I'm wondering if you ready something in the docs that I may have missed?

Reply:
https://docs.mongodb.com/manual/aggregation/#aggregation-pipeline-example in this example, they said that `$match` was used to filter out the documents that we don't want the pipeline to operate on. We want to operate on every rating document, so it didn't seem like there was any need for the `$match` operator.

Reply:
Fantastic!